<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>280</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>300</width>
    <height>256</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>:: KCpuFreq 0.2 ::</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelCPUs">
      <property name="text">
       <string>CPU(s):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listCPUs">
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <item>
       <property name="text">
        <string>All cores</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>core 0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>core 1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>core 2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>core 3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>core 4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>core 5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>core 6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>core 7</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="labelGovernor">
        <property name="text">
         <string>Governor: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxGovernor">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>30</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>performance</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ondemand</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="buttonSet">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set governor for the selected CPU(s)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>&amp;Set</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonClose">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Close this window&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>&amp;Close</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>&amp;Menu</string>
    </property>
    <addaction name="actionCpufreq"/>
    <addaction name="actionCpupower"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="actionCpufreq">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;cpufreq</string>
   </property>
  </action>
  <action name="actionCpupower">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>cpu&amp;power</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonClose</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>35</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
